src/ch04-00-understanding-ownership.md

```diff
@@ -1,7 +1,6 @@
-# Understanding Ownership
+# ทำความเข้าใจเกี่ยวกับ Ownership
 
-Ownership is Rust’s most unique feature, and it enables Rust to make memory
-safety guarantees without needing a garbage collector. Therefore, it’s
-important to understand how ownership works in Rust. In this chapter, we’ll
-talk about ownership as well as several related features: borrowing, slices,
-and how Rust lays data out in memory.
+Ownership เป็นลักษณะเฉพาะตัวที่สุดของ Rust ที่ทำให้ Rust สามาถรับประกัน memory safety ได้โดยที่ไม่ต้องมี
```

Comment:
```suggestion
Ownership เป็นลักษณะเฉพาะตัวที่สุดของ Rust ที่ทำให้ Rust สามารถรับประกัน memory safety ได้โดยที่ไม่ต้องมี
```

Reply:
ผมชอบคำว่าฟีเจอร์แฮะ คิดว่าทับศัพท์ไปก็ยังเข้าใจ

Reply:
@dodadoa I like the word ฟีเจอร์ as well. Should I use the transliteration for this word?

Reply:
@kkrittipong Thank you! Already commit your suggested change.

Reply:
@kkrittipong What do you think? Should it be `ลักษณะเฉพาะ` or `feature` or `ฟีเจอร์`. IMHO, I like `feature` the most.

Reply:
ฟีเจอร์ seem blending to thai already, so I think it's ok

Reply:
Let's have a majority vote for this, @kkrittipong  we need you to be the one who decide which word should we use? `feature` or `ฟีเจอร์`. And we gonna continue using it in the entire book.

Reply:
ผมเลือกฟีเจอร์ครับเพราะอ่านแล้วเข้าใจได้เลยไม่จำเป็นต้องใช้คำภาษาอังกฤษ

Reply:
Done! Please help to review the PR again! @dodadoa @kkrittipong